### PR TITLE
Lopland holobadges in the loadout & security vendor! Also fixes them for detectives! 

### PR DESCRIPTION
**Pull request by Plum-but-gayer**
## About The Pull Request
Adds both types of holobadge (normal and with lanyard) to the neck slot loadout menu, locked for security jobs only. Also changes the access you need to swipe your ID on them from brig to general sec, allowing detectives to finally use their holobadges. Resprites the non-lanyard holobadge to be an appropriate Lopland blue rather than red. Finally, a box of holobadges can be purchased from the Armadyne vendor in security. 

## How This Contributes To The Skyrat Roleplay Experience

Holobadges have always been a cool little flavor item that has disappeared over time for one reason or another. Keeping them around and easy to access is a low-cost way of giving security more ways to RP and drag Lopland lore in from the realm of google documents into the world of actual inround stuff. Also detectives should be able to use their own badges. :)

## Proof of Testing

Runs fine when I run it on my machine. 

<details>
<summary>Screenshots/Videos</summary>
image

image

image

image

image

</details>

## Changelog

:cl:
add: Overbearing authority figures rejoice! Holobadges are now available in the neck loadout slot and the Armadyne peacekeeper vendor, now in a pleasant Lopland blue.
fix: Detectives can now actually swipe their ID on their holobadges.
/:cl:

### CODE REVIEW

**Comment by Namwty** [comment]
Can't wait to wake up in a bar as a detective and lie to my lopland buddies that I did not lose my badge ( or gun, for that matter )

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 		if(istype(object, /obj/item/card/id))
 			id_card = object
 
-		if(ACCESS_BRIG in id_card.access || (obj_flags & EMAGGED))
+		if(ACCESS_SECURITY in id_card.access || (obj_flags & EMAGGED))
```
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/clothing/under/accessories.dm:101]: Correct me if I'm wrong, I haven't memorized access requirements for every job, but wouldn't this mean theoretically Dept. Guards could use them?

**Comment by theselfish** [comment]
Thoughts on adding a box or two of them to the Equipment Vendor?

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm (hunk 1 above)
Comment (Plum-but-gayer) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm:101]: extremely possible, idk if that would be a good thing or bad thing tbh. I'm not sure what access detectives have that dept. guards don't, though

**Comment by Plum-but-gayer** [comment]
> Thoughts on adding a box or two of them to the Equipment Vendor?

good idea, happy to add

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm (hunk 1 above)
Comment (theselfish) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm:101]: Very fair point.
Not exactly a major, PR breaking bug. The worst that'll happen is a departmental guard gets one and...can flash it?

File: modular_skyrat/master_files/icons/mob/clothing/accessories.dmi
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/icons/mob/clothing/accessories.dmi:1]: One of the dirs of the lopland badge here is still red.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
That was the only nitpick I had about this, should be good to merge once fixed!

File: modular_skyrat/master_files/icons/mob/clothing/accessories.dmi
Comment (Plum-but-gayer) [reply modular_skyrat/master_files/icons/mob/clothing/accessories.dmi:1]: image

Maybe I don't know how dirs work, but this one has been changed to blue! Is there something I'm missing?

Comment (Plum-but-gayer) [reply modular_skyrat/master_files/icons/mob/clothing/accessories.dmi:1]: OH i found it. so tiny lol

**Comment by Plum-but-gayer** [comment]
not sure why I'm failing the PR labeller test

**Comment by vinylspiders** [comment]
> not sure why I'm failing the PR labeller test

PR labeler action got an update and had to be updated on our end.

**Comment by Plum-but-gayer** [comment]
should be good to merge now btw 👍

**Review by vinylspiders — APPROVED** [review]
Looks fine to me, @GoldenAlpharex sprites look ok to you?

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me!

**Comment by GoldenAlpharex** [comment]
Just fixed the conflict, ready to merge once CI passes again.

**Pull request merged**